phy/io/kwik_model.py

```diff
@@ -97,6 +98,17 @@ def _concatenate_spikes(spikes, recs, offsets):
     return (spikes + offsets[recs]).astype(np.uint64)
 
 
+def list_kwik(folders):
+    """ return the list of kwik files found in a list of folders """
```

Comment:
* Remove spaces between """ and word
* Uppercase for the first letter
* Dot at the end of the sentence
* First letter of Kwik in uppercase